src/main/java/com/facebook/openwifirrm/ucentral/models/AggregatedState.java

```diff
@@ -201,16 +196,28 @@ public boolean equals(Object obj) {
 			Objects.equals(radio, other.radio);
 	}
 
+	/**
+	 * Check whether some other AggregatedState and this one are matached for aggregation.
```

Comment:
`Check whether some other AggregatedState and this one are matached for aggregation` -> `Check whether the passed-in AggregatedState and this one match for aggregation`